wazuh/wazuh-dashboard-plugins
# Fix error when using chinese characters in username

### Description
This PR aims to solve an issue detected when the Dashboard username contains chinese characters. 
 
### Issues Resolved
#6068 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/42900763/60d691e7-a3f6-4826-a6ea-b258645e90e2)

### Test
Using an environment with a real manager:
- Create an internal user with chinese characters. For example `Test 证明`. Assign admin permissions
- Map the internal user with the admin role in the Wazuh Plugin
- Log in with the new user
- Navigate in the app and verify that no errors are shown and the app works as expected

### Check List
